packages/ui/components/input-stepper/src/LionInputStepper.js

```diff
@@ -427,7 +441,9 @@ export class LionInputStepper extends LocalizeMixin(LionInput) {
   /** @protected */
   _inputGroupTemplate() {
     return html`
-      <div class="input-stepper__value">${this.__valueText}</div>
+      <output for="${this.fieldName}" data-self-destruct="2000" class="input-stepper__value"
```

Comment:
The `for` should point to the input field.

```suggestion
      <output for="${this._inputId}" data-self-destruct="2000" class="input-stepper__value"
```